tensorflow/compiler/xla/debug_options_flags.cc

```diff
@@ -50,6 +50,7 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
   opts.set_xla_dump_max_hlo_modules(-1);
   opts.set_xla_dump_module_metadata(false);
   opts.set_xla_dump_hlo_as_long_text(false);
+  opts.set_xla_dump_disable_mlir_pretty_form(false);
```

Comment:
I think it is easier to reason about without the double negation here.
Why not name the flag set_xla_dump_enable_mlir_pretty_form and set it to true by default?

Reply:
sounds good to me, changed it to "enable"